lib/src/components/NxTransferList/NxTransferList.tsx

```diff
@@ -137,7 +139,7 @@ export default function NxTransferList<T extends string | number>(props: Props<T
           }
           <div className="nx-transfer-list__item-list">
             { visibleAvailableItems.map(
-                i => <TransferListItem key={i.id} checked={false} onChange={onChange} { ...i } />)
+                i => <TransferListItem<T> key={i.id} { ...{ checked: false, onChange, ...i } } />)
```

Comment:
Why this change?

Reply:
> Why this change?

I was getting `is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes` error and that fixed it.
Now that we're removing propTypes from TransferListItem it looks like it's working again. Going to change it back.

Reply:
> Why this change?

This was changed back in this commit:
0f0c1c1799991ec9d4c9e64eca29b9c86c4061c0